test/telegraf.js

```diff
@@ -1,6 +1,12 @@
 const test = require('ava')
 const { Telegraf, session } = require('../')
 
+function createBot (...args) {
+  const bot = new Telegraf(...args)
+  bot.botInfo = { id: 42, is_bot: true, username: 'bot', first_name: 'Bot' }
+  return bot
+}
+
```

Comment:
Can and should we reuse this from `composer.js`? I did not find a way to make testing modules dependent on a third factory module without ava complaining that there were no tests in it.

Reply:
I wouldn't bother. I'm surprised you didn't have to specify more properties in `botInfo`.